com.authy.Authy.yaml

```diff
@@ -0,0 +1,65 @@
+app-id: com.authy.Authy
+
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+
+tags:
+  - proprietary
+
+command: authy
+finish-args:
+  - --socket=x11
+  - --device=dri
+  - --share=ipc
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+
+modules:
+  - name: unsquashfs
+    buildsystem: simple
+    build-commands:
+      - XZ_SUPPORT=1 make -C squashfs-tools -j ${FLATPAK_BUILDER_N_JOBS} unsquashfs
+      - install -Dpm755 -t "${FLATPAK_DEST}/bin" squashfs-tools/unsquashfs
+    sources:
+      - type: git
+        url: https://github.com/plougher/squashfs-tools.git
+        tag: '4.5.1'
+        commit: afdd63fc386919b4aa40d573b0a6069414d14317
+
+  - name: authy
+    buildsystem: simple
+    build-commands:
+      - install -Dpm755 -t "${FLATPAK_DEST}/bin" apply_extra
+      - install -Dpm755 -t "${FLATPAK_DEST}/bin" authy
+    post-install:
+      - install -Dpm644 -t "${FLATPAK_DEST}/share/icons/hicolor/512x512/apps" com.authy.Authy.png
+      - install -Dpm644 -t "${FLATPAK_DEST}/share/applications" com.authy.Authy.desktop
+      - install -Dpm644 -t "${FLATPAK_DEST}/share/metainfo" com.authy.Authy.metainfo.xml
+    sources:
+      - type: extra-data
+        filename: authy.snap
+        url: https://api.snapcraft.io/api/v1/snaps/download/H8ZpNgIoPyvmkgxOWw5MSzsXK1wRZiHn_12.snap
+        sha256: bd26ea4f6b3994b114d52483682fac4baaade38ea248a61480ab9d3b958c3aee
+        size: 109084672
+        x-checker-data:
+          type: snapcraft
+          name: authy
+          channel: stable
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - unsquashfs -dest authy -no-progress -excludes authy.snap meta usr lib command.sh desktop-init.sh desktop-common.sh desktop-gnome-specific.sh
+          - rm -f authy.snap
+      - type: script
+        dest-filename: authy
+        commands:
+          - exec /app/extra/authy/authy --no-sandbox "$@"
```

Comment:
Have you checked if it works without `--no-sandbox` when run with [Zypak](https://github.com/refi64/zypak)?

Reply:
I tried and with Zypak and XWayland at least, and it seams it don't render the window at all. With wayland I didn't managed to start the application at all with the chromium ozone hint auto options.
Maybe a future release wayland will work.

Also their official snap pakage uses the no sandbox option for now.

Reply:
Fair, thanks.